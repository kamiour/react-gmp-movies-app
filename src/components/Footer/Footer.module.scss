@import 'src/scss/variables';

.footer {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 70px;
  background-color: $color_charcoal;
}