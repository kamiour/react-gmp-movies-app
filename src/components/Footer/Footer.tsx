import { PropsWithChildren } from 'react';
import styles from './Footer.module.scss';

const Footer = ({ children }: PropsWithChildren<{}>) => <footer className={styles.footer}>{children}</footer>;

export default Footer;
