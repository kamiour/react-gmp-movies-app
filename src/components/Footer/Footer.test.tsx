import { render } from '@testing-library/react';
import Footer from './Footer';

describe('Footer', () => {
  it('should match Footer snapshot', () => {
    const { asFragment } = render(<Footer>Footer</Footer>);
    expect(asFragment()).toMatchSnapshot();
  });
});
