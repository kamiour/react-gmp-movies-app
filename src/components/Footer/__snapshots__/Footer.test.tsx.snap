// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match Footer snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    Footer
  </footer>
</DocumentFragment>
`;
